### PR TITLE
[FIX] website_sale_delivery: update shopping cart

### PR DESCRIPTION
**Pull request by simongoffin**
The delivery must be updated when the cart is updated.
Related to the task : 12239
opw:641913

### CODE REVIEW

File: addons/website_sale_delivery/models/sale_order.py [hunk 1]
```diff
@@ -123,3 +123,16 @@ def _get_website_data(self, cr, uid, order, context=None):
...
+
+        if self.pool['sale.order.line'].browse(cr, uid, [line_id], context=context).exists():
+            values = super(SaleOrder, self)._cart_update(
+                cr, uid, ids, product_id, line_id, add_qty, set_qty, context, **kwargs)
+
+        if add_qty or set_qty is not None:
+            for sale_order in self.browse(cr, uid, ids, context=context):
+                self._check_carrier_quotation(cr, uid, sale_order, context=context)
+
+        return values
```
Comment (Gorash) [inline addons/website_sale_delivery/models/sale_order.py]: if no values ?

**Comment by simongoffin** [comment]
@beledouxdenis

**Comment by Gorash** [comment]
Ok for me. (may be add a comment)

**Comment by beledouxdenis** [comment]
Seems rather good for me too.

**Pull request closed without merging**